src/ROUTES.js

```diff
@@ -54,6 +54,11 @@ export default {
     SETTINGS_CONTACT_METHOD_DETAILS: `${SETTINGS_CONTACT_METHODS}/:contactMethod/details`,
     getEditContactMethodRoute: contactMethod => `${SETTINGS_CONTACT_METHODS}/${encodeURIComponent(contactMethod)}/details`,
     SETTINGS_NEW_CONTACT_METHOD: `${SETTINGS_CONTACT_METHODS}/new`,
+    SETTINGS_TWO_FACTOR_IS_ENABLED: 'settings/security/two-factor-auth/enabled',
```

Comment:
This one I don't agree because the screens have different purposes. The `IsEnabled` page is not where you enable 2-FA, it only informs the user that 2-FA is enabled and allows the user to disable. 2-FA is really enabled in the `CodesPage`